### PR TITLE
Added Chuming's slides and photo

### PR DESCRIPTION
**Pull request by changchuming**
Ah. I'll update the slides to Hackware 2.4. Should NTU OSS still be in there?

### CODE REVIEW

**Comment by notthetup** [comment]
Can we add a slide for Venue and Food Sponsor: NUS Enterprise and this logo.

nus_enterprise logo_vertical

File: index.html [hunk 1]
```diff
@@ -86,12 +90,14 @@
 						<img src="img/km.jpg" alt="Kheng Meng" />
 						<p>khengmeng</p>
 					</li>
+					<li>
+						<img src="img/chuming.jpg" alt="Kheng Meng" />
+						<p>chuming (guest mc)</p>
```
Comment (sayanee) [inline index.html:95]: @changchuming `alt` tag should be your name?

**Review by sayanee — CHANGES_REQUESTED** [review]
Thanks @changchuming! Let's also amend the slide on the agenda following the facebook event page 😄

File: index.html (hunk 1 above)
Comment (changchuming) [reply index.html:95]: Yeah my bad haha.

Comment (changchuming) [inline index.html]: Didn't realise that the content was for the previous Hackware! Will change it ASAP.

Comment (changchuming) [inline index.html]: So should the NTU OSS logo still be there?

**Pull request closed without merging**